mmedit/models/editors/disco_diffusion/guider.py

```diff
@@ -15,8 +14,11 @@
 from torchvision import __version__ as TORCHVISION_VERSION
 
 from mmedit.models.losses import tv_loss
+from mmedit.utils import try_import
 from .secondary_model import alpha_sigma_to_t
 
+clip = try_import('clip')
```

Comment:
I suggest we should add `assert clip is not None` in `ImageTextGuider.__init__` since this class is built upon `clip`.